pkg/acs/lib/err.c

```diff
@@ -19,46 +17,4 @@ void errchk() {
 	fflush (stdout);
 	status = 1;
 	}
-}
-
-void asnkwerr (char *keyword, char *file) {
-	sprintf (MsgText, "Keyword \"%s\" not found in %s", keyword, file);
-	asnerror (MsgText);
-}
-
-void asnopenerr (char *name) {
-	sprintf (MsgText, "Can't open file %s", name);
-	asnerror (MsgText);
-}
-
-void asnreaderr (char *name) {
-	sprintf (MsgText, "Can't read file %s", name);
-	asnerror (MsgText);
-}
-
-void asnfilerr (char *name) {
-	sprintf (MsgText, "while trying to read file %s", name);
-	asnerror (MsgText);
-}
-
-void asnwarn (char *message) {
-
-	char line[CHAR_LINE_LENGTH+1];
-	
-	/* Use macro for beginning of Warning message */
-	sprintf(line,"%s",WARN_PREFIX);
-	strcat (line,message);
-
-	printfAndFlush(line);
-}
-
-void asnerror (char *message) {
-		
-	char line[CHAR_LINE_LENGTH+1];
-	
-	/* Use macro for beginning of Warning message */
-	sprintf(line,"%s",ERR_PREFIX);
-	strcat (line,message);
-
-	printfAndFlush(line);
-}
+}
```

Comment:
Remove asnwarn and asnerror from acs/include/acs.h?

Reply:
Done